programming_examples/basic/vector_vector_add/vector_vector_add.py

```diff
@@ -7,32 +7,64 @@
 # (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
 import numpy as np
 import sys
+import argparse
 
 from aie.iron import ObjectFifo, Program, Runtime, Worker
 from aie.iron.placers import SequentialPlacer
 from aie.iron.device import NPU1Col1, NPU2Col1, XCVC1902
 from aie.iron.controlflow import range_
 
+import aie.iron as iron
 
-def my_vector_add():
-    N = 256
-    n = 16
-    N_div_n = N // n
+# The JIT-compiled kernel relies on inputs from the command line.
+# Because of that, we need to parse the arguments before the JIT is invoked.
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-v", "--verbose", action="store_true", help="Enable verbose output"
+)
+parser.add_argument(
+    "-d",
+    "--device",
+    choices=["npu", "npu2", "xcvc1902"],
```

Comment:
OK to deprecate xcvc1902 (vck5000) for this example.
```suggestion
    choices=["npu", "npu2"],
```